coalib/bearlib/abstractions/Linter.py

```diff
@@ -695,6 +740,11 @@ def linter(executable: str,
         A custom message that gets displayed when ``check_prerequisites``
         fails while invoking ``prerequisite_check_command``. Can only be
         provided together with ``prerequisite_check_command``.
+    :param global_bear:
+        Wether the created bear should be a global bear or not. Global bears
+        will be run only once instead of once per file. The file and filename
```

Comment:
maybe enclose `file` and `filename` also in double backticks, as you refer to them as arguments^^